webapp/tests/karma/unit/store/index.js

```diff
@@ -26,6 +45,26 @@ describe('Store', function() {
     return { global: state };
   }
 
+  function createAnalyticsState(state) {
```

Comment:
Feel free to use lambdas fully in test code since it doesn't run through uglify/browserify so doesn't have the same issues with the sourcemap issues.